proto/engine/v1/execution_engine.proto

```diff
@@ -75,7 +75,7 @@ message ExecutionPayload4844 {
 	bytes base_fee_per_gas      = 12 [(ethereum.eth.ext.ssz_size) = "32"];
 	bytes block_hash            = 13 [(ethereum.eth.ext.ssz_size) = "32"];
 	repeated bytes transactions = 14 [(ethereum.eth.ext.ssz_size) = "?,?", (ethereum.eth.ext.ssz_max)  = "1048576,1073741824"];
-	uint64 excess_blobs         = 15; // New in EIP-4844
+	uint64 excess_data_gas      = 15; // New in EIP-4844
```

Comment:
I guess this should probably be 32 byte byte-field to be compliant with the spec?

Reply:
This is fixed.